infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/KubernetesServerResolver.java

```diff
@@ -87,12 +85,14 @@ private void fillServiceServers(Service service, Map<String, ServerImpl> servers
             (name, config) ->
                 servers.put(
                     name,
-                    newServer(
-                        config.getProtocol(),
-                        service.getMetadata().getName(),
-                        config.getPort(),
-                        config.getPath(),
-                        config.getAttributes())));
+                    new RuntimeServerBuilder()
+                        .protocol(config.getProtocol())
+                        .host(service.getMetadata().getName())
+                        .port(config.getPort())
+                        .path(config.getPath())
+                        .attributes(config.getAttributes())
+                        .targetPort(config.getPort())
```

Comment:
Should be `targetPort` be renamed here to `port` as well?

Reply:
I'll rename it

Reply:
Actually there is `port` method which is needed to build real URL of the server. I'll leave targetPort here